app/models/kudo.rb

```diff
@@ -3,6 +3,7 @@
 class Kudo < ApplicationRecord
   belongs_to :giver, class_name: 'Employee'
   belongs_to :receiver, class_name: 'Employee'
+  belongs_to :company_value, class_name: 'CompanyValue'
```

Comment:
you do not need class_name in this case - rails will find it by itself

```suggestion
  belongs_to :company_value
```

Reply:
ok, that's logical actually. I corrected it